bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/SPCoordinates.scala

```diff
@@ -98,9 +98,15 @@ final class SPCoordinates(var coordinates: Coordinates) extends SPSkyObject {
 
   override def getDecString(time: GOLong): GOption[String] =
     Some(coordinates.dec.toAngle.formatHMS).asGeminiOpt
+
+  override val getName: String = SPCoordinates.Name
+
+  override def setName(name: String): Unit =
+    throw new UnsupportedOperationException("Cannot set name of coordinates")
```

Comment:
Perhaps `setName` shouldn't be in the `SPSkyObject` interface if it doesn't apply?